### DISCUSSION TITLE
Databases did not upgrade when upgrading from 5.3 to 6.2

### DISCUSSION THREAD

**Discussion by seanrockster** [ravendb/ravendb#19501] [General]
Hi

I followed the documentation and removed all binary files, however there are lots of folders in the server folder so I removed that as well and left only the RavenData and settings.json. I copied the 6.2 files and ran the Raven.Server.exe but none of the existing databases were upgraded at least they do not show up in the 6.2 version. Any ideas what I can checked?

Thanks

**Comment by ppekrol** [comment]
Hi @seanrockster 

is there a chance that you removed System folder? It contains system storage with meta information about databases and their paths. That could explain why you do not see any database in the Studio.

**Comment by seanrockster** [comment]
@ppekrol Hi, no, i made sure the system folder was intake. I basically removed everything except for that folder so it had the existing database folder and system folder.

**Reply by ppekrol** [reply]
Can you paste your settings.json (please remove any confidential info) ? what are you getting from the `/databases` endpoint?

**Reply by seanrockster** [reply]
this is running locally:

{
  "DataDir": "RavenData",
  "License.Eula.Accepted": true,
  "Setup.Mode": "Unsecured",
  "Security.UnsecuredAccessAllowed": "PublicNetwork",
  "ServerUrl": "http://127.0.0.1:54889",
  "ServerUrl.Tcp": "tcp://127.0.0.1:38888"
}

/databases:

{"Databases":[]}

**Reply by ppekrol** [reply]
In RavenData folder do you see System folder and beside that other Databases? Can you check folder creation date for System?

As an emergency operation, if you will create a new database in the same path as previous it will be recovered (without any cluster data e.g. compare exchange).
